app/Http/Controllers/Api/TalksController.php

```diff
@@ -0,0 +1,26 @@
+<?php namespace Symposium\Http\Controllers\Api;
+
+use Illuminate\Support\Facades\Auth;
+use LucaDegasperi\OAuth2Server\Facades\AuthorizerFacade as Authorizer;
+use Symposium\ApiResources\Talk;
+use User;
+
+class TalksController extends BaseController
+{
+    public function show($id)
+    {
+        $user = User::find(Authorizer::getResourceOwnerId());
```

Comment:
I think you might as well `findOrFail` on these, even though you are basically guaranteed the user is there barring some weird database catastrophe. If something goes wrong you'l at least get a more useful error than "Cannot call method talks() on non-object".